translations/messages.en.yml

```diff
@@ -1,118 +1,33 @@
-service_provider: Service provider
+# Registration
 
-page.title: GSSP SDK
+registration.sub_title: Registration
+status_registration_initial: Registrate with your security key.
 
-page.header.stepup-gateway-gssp: Wiki
-page.header.stepup-gssp-bundle: GSSP-Bundle
+# Authentication
 
-page.navigation.route.homepage: Home
-page.navigation.route.app_identity_registration: Registration
-page.navigation.route.app_identity_authentication: Authentication
-page.navigation.route.gssp_saml_sso: SSO
-page.navigation.route.gssp_saml_sso_return: SSO return
-page.navigation.route.gssp_saml_metadata: Metadata
-page.navigation.route.sp_demo: Demo SP
+authentication.sub_title: Authentication
+status.authentication_initial: Authenticate with your security key.
 
-page.footer.terms: Terms and Conditions
+# General status messages
 
-page.index.title: Welcome to the Generic SAML Stepup Provider SDK
-page.index.body.intro: |
-  The application is ready for use. This example project can be used to develop your own implemention of a GSSP verification method.
-  It demonstrates registration and authentication of second factor tokens initiated by a dummy service provider.
+status.general_error: Oops! something wrong happend, please try again.
+status.webauthn_not_supported: Webauthn is not supported by your browser.
```

Comment:
WebAuthn is the name of the game. Consider updating all mentions of `Webauthn` with `WebAuthn`, unless the PO requested otherwise.

Reply:
👍